specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json

```diff
@@ -1166,6 +1169,7 @@
         },
         "percentComplete": {
           "type": "integer",
+          "format": "int64",
```

Comment:
Hi @ChenTanyi , This change will not impact customers as we are just adding the int format to fix linter error.